### DISCUSSION TITLE
Suporte a logon da conta CNPJ

### DISCUSSION THREAD

**Discussion by tarciziovn** [andreroggeri/pynubank#357] [Ideas]
Alguém sabe se é possível fazer logon com as credenciais de conta CNPJ através da API? Só vi referencia de credenciais passando como argumento login=CPF
Não encontrei alternativas para uso de CNPJ

Se a request for a mesma passando os digitos do CPF ou CNPJ como argumento, sugiro alterar o nome dos parâmetros para apenas "login" para de forma intuitiva atender os dois casos.

**Comment by andreroggeri** [comment]
Já tivemos algumas discussões sobre isso aqui, mas nunca foi pra frente.

Atualmente a lib não funciona com a conta PJ.

#304 #300 #297

**Reply by tarciziovn** [reply]
Poxa, neste caso se nao for assim não serve :/
Acabei descobrindo que no BB tem API para PJ... acho lamentável o próprio Nubank não fornecer API equivalente, nem que cobrasse pelo serviço para garantir a disponibilidade do serviço e evitar o abuso no disparo dos requests com base na quantidade de operações mensais.

**Reply by pecampelo** [reply]
Acredito que possamos continuar a investigar isso, irei eu pessoalmente, já que agora a conta PJ do Nubank foi lançada sem precisar ter uma correlação direta com a entidade da conta PF. @tarciziovn

**Reply by andreroggeri** [reply]
Sim @tarciziovn acho bem ruim isso. Tudo que foi implementado aqui foi pegando os requests que o site e o app fazem, mas não tem garantia que isso vai funcionar pra sempre (Embora já esteja aí há uns 5 anos hehe).

**Reply by andreroggeri** [reply]
@pecampelo eu já tentei dar uma olhada nisso com outras pessoas. Me confirma uma coisa, sempre que você loga na sua conta PJ pela primeira vez, você precisa fazer a autenticação com a selfie? Na época que eu olhei era assim, e não valia a pena pra mim tentar fazer a engenharia reversa disso.

**Reply by pecampelo** [reply]
eu também já tentei. particularmente, não tentei esta possibilidade, só to trazendo pra disscussão. 

Vi a notícia de que eles desassociaram as entidades recentemente, provavelmente irei averiguar num fim de semana deste mês.
